plugins/woocommerce/src/Internal/Admin/Settings/PaymentsController.php

```diff
@@ -75,7 +76,13 @@ public function adjust_feature_default_enablement( $option_name, $option_value )
 			return;
 		}
 
-		// Make sure the feature is disabled by default for existing stores.
+		$wc_initial_installed_version = get_option( \WC_Install::INITIAL_INSTALLED_VERSION, '0.0.0' );
+		// If the WooCommerce installed version is 9.7+, we don't need to do anything. Just let it be enabled.
+		if ( version_compare( $wc_initial_installed_version, '9.7.0', '>=' ) ) {
```

Comment:
All stores created with 9.7 that didn't have user interactions with the WCAdmin feature flag, should get the core feature enabled.